lua/lspconfig/server_configurations/pylyzer.lua

```diff
@@ -0,0 +1,36 @@
+local util = require('lspconfig').util
+
+local bin_name = 'pylyzer'
+local cmd = { bin_name, '--server' }
+
```

Comment:
check windows env here. in windows you need `cmd.exe`

Reply:
No clue how Windows runs executables but this seems to be how other configurations handle it and seems reasonable after reading the cmd.exe documentation.